src/client/coll/results.rs

```diff
@@ -0,0 +1,104 @@
+use bson;
+use bson::Bson;
+
+use std::collections::BTreeMap;
+
+/// Results for a bulk write operation.
+#[derive(Clone)]
+pub struct BulkWriteResult {
```

Comment:
I think we'll keep 'Result', that's something that the CRUD spec isn't very flexible on unfortunately